kubernetes/core-svc.yml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: busybox-svc  
+  name: core-app-svc  
   labels:
-    app: busybox
+    app: core-app
 spec:
   type: ClusterIP
   ports:
     - protocol: TCP
```

Comment:
`TCP` protocols smells to me when we want to use REST api. If you know what I mean.

Reply:
Sorry, my bad. I didn't realized the protocol on the right level of TCP/IP model. Solved.